spec/features/comment_types/1_user_views_index_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'view comment_types#index', %{
+  As a Guest
+  I want to view the comment_types#index
+  So I can pick from all comment_types.
+  } do
+  # Acceptance
+  # When a guest visits the comment_types#index_path,
+  # they should expect to see a list of all comment_types
+  # When a guest clicks the view button on the comment form,
+  # they should expect to see a list of all comment_types
+  # The list should be sorted alphabetically by Title
+  # The list should include the Name
+
+  let!(:type1) { FactoryGirl.create(:variation) }
+  let!(:type2) { FactoryGirl.create(:comment_type) }
+  let!(:type3) { FactoryGirl.create(:style_point) }
+  let!(:dance) { FactoryGirl.create(:dance) }
+
+  scenario 'sucessfully view the comment_types#index' do
+    visit comment_types_path
+
+    comment_types = page.all("div#comment_typelist ul li")
+
+    expect(comment_types[0]).to have_content(type2.name)
+    expect(comment_types[1]).to have_content(type3.name)
+    expect(comment_types[2]).to have_content(type1.name)
+  end
+
+  pending 'sucessfully view the comment_types#index after click on
+  the comment form view button' do
+    visit new_dance_dance_comment_path(dance)
+    click "View Types"
+
+    expect(page).to have_current_path(comment_types_path)
+
+    # IN A POPUP WINDOW MAYBE WHEN ? ICON IS CLICKED INSTEAD OF
+    # "VIEW COMMENT TYPES"
+  end
```

Comment:
Maybe the list of comment_types wants to be a `<select>` dropdown. Or if not:

From a user experience perspective, it might be cool to have a button on the `new_dance_dance_comment_path` that makes a list of the comment_types appear on the same page via JS. Or instead of a list perhaps a tooltip. This will only work if there are say a dozen or so comment_types

Not sure what the user experience intent here, but if you want to explore options feel free to reach out on slack and we can have a convo